### PR TITLE
Batch of fixes from the Lancaster doc review

### PR DESCRIPTION
**Pull request by ganicke**

### CODE REVIEW

**Comment by ganicke** [comment]
Re: https://github.com/Microsoft/pxt/issues/2416

File: libs/core/basic.cpp [hunk 1]
```diff
@@ -85,6 +85,8 @@ namespace basic {
      * @param interval time in milliseconds between each redraw
      */
     //% help=basic/show-animation imageLiteral=1 async
+    //% blockId=device_show_animation 
+    //% block="show animation"
```
Comment (pelikhan) [inline libs/core/basic.cpp]: This API is not well supported in blocks. Needs testing.

Comment (ganicke) [inline libs/core/basic.cpp]: Ok, I'll take block support off. Leave it or remove from the basic.md card section?

**Comment by pelikhan** [comment]
Leave it, works in TypeScript

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**